### PR TITLE
feat: add support for create database

### PR DESCRIPTION
**Pull request by cvng**
## What kind of change does this PR introduce?

add support for `create database`

## What is the current behavior?

panics

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

**Comment by cvng** [comment]
not sure if we wait for #87 to have the snapshot in this PR

File: crates/parser/tests/snapshots/statements/valid/0047@1.snap [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          Owner@18..23 "OWNER"
+          Whitespace@23..24 " "
+          Ident@24..27 "abc"
+        Whitespace@27..28 " "
+        DefElem@28..38
+          Connection@28..38 "CONNECTION"
+        Whitespace@38..39 " "
+        Limit@39..44 "LIMIT"
+        Whitespace@44..45 " "
+        Iconst@45..46 "5"
```
Comment (psteinroe) [inline crates/parser/tests/snapshots/statements/valid/0047@1.snap:24]: These should be part of DefElement

File: crates/parser/tests/snapshots/statements/valid/0047@4.snap [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        Create@1..7 "CREATE"
+        Whitespace@7..8 " "
+        Database@8..16 "DATABASE"
+        Whitespace@16..17 " "
+        Ident@17..18 "x"
+        Whitespace@18..19 " "
+        DefElem@19..27
+          Location@19..27 "LOCATION"
+        Whitespace@27..28 " "
+        Default@28..35 "DEFAULT"
```
Comment (psteinroe) [inline crates/parser/tests/snapshots/statements/valid/0047@4.snap:18]: Should also be part of DefElement

**Review by psteinroe — CHANGES_REQUESTED** [review]
Can you use the parent of DefElement to fix this?

File: crates/parser/tests/snapshots/statements/valid/0047@1.snap (hunk 1 above)
Comment (cvng) [reply crates/parser/tests/snapshots/statements/valid/0047@1.snap]: thanks, fixed in 45f88be683ba728c8454d02c6a2feb1349f3ce58

File: crates/parser/tests/snapshots/statements/valid/0047@4.snap (hunk 2 above)
Comment (cvng) [reply crates/parser/tests/snapshots/statements/valid/0047@4.snap]: fixed in 45f88be683ba728c8454d02c6a2feb1349f3ce58

**Review by psteinroe — APPROVED** [review]
lgtm, thanks!

**Pull request merged**